src/sentry/options/defaults.py

```diff
@@ -391,3 +391,7 @@
 # contents are a list of project IDs to message types to be randomly assigned
 # e.g. [{"project_id": 2, "message_type": "error"}, {"project_id": 3, "message_type": "transaction"}]
 register("kafka.send-project-events-to-random-partitions", default=[])
+
+# Mechanism for dialing up the last-seen-updater, which isn't needed outside
+# of SaaS (last_seen is a marker for deleting stale customer data)
+register("sentry-metrics.last-seen-updater.accept-rate", default=0.0)
```

Comment:
I think you still need to send a first PR to register the option and a separate PR (after that one is deployed) to actually use it. When you deploy the option you need to ask the ops team to set it in cache. 
Then you can deploy something that uses it. But I may not be up to date on this process. Please verify.

Reply:
good call, I'll make a separate PR for that, though last time I asked ops to initialize it I think they said that's not a thing anymore (but I still think we are supposed to deploy the new option independently)

Reply:
config-specific PR: https://github.com/getsentry/sentry/pull/35168